templates/layouts/configs/_save-sign-out-button.html

```diff
@@ -17,7 +17,8 @@
                 "data-ga-category": "Navigation",
                 "data-ga-action": "Save and sign out click"
             },
-            "buttonStyle": "exit"
```

Comment:
Is `buttomStyle` still a valid use in the DS?

Reply:
It is for some button types but not for exit anymore. For exit we just add the icon, icon position and variant now